### PR TITLE
added plotColGraph and plotRowGraph function

### PR DESCRIPTION
**Pull request by FelixErnst**

### CODE REVIEW

**Comment by microsud** [comment]
@FelixErnst the plots look cool and function works well. I assume that the process of creating the graph table will be shown in `MiaBook`? 
One more point, in `plotRowGraph`
```
plotRowGraph(row_graph,
             genus,
             colour_by="Kingdom",
             edge_colour_by="weight",
             edge_width_by="weight",
             show_label=TRUE*) 
```
*this is tricky because too many points to label. Would it be possible to have a user defined cut-off for labels? For.eg. 
if a node has more than three or four edges with weights more than a give value, that node is labelled? Something similar to the prevalence option, i.e. detection atleast 0.001 in 50% samples.

File: R/plotGraph.R [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+#' @param name If \code{x} is a 
+#' \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#' the key for subsetting the \code{metadata(x)} to a graph object.
+#' 
+#' @param show_label logical scalar or character vector. Should tip labels
+#'   be plotted or if a logical vector is provided, which labels should be 
+#'   shown? Only values corresponding to actual labels will be plotted. 
+#'   (default: \code{show_label = FALSE})
+#' 
+#' @param add_legend logical scalar. Should legens be plotted?
```
Comment (microsud) [inline R/plotGraph.R:22]: legens -> legends

File: R/miaViz.R [hunk 2]
```diff
@@ -14,7 +14,42 @@ NULL
...
 #' @importFrom rlang sym !! :=
 #' @importFrom dplyr %>%
 #' @importFrom BiocGenerics ncol nrow
 NULL
+
+#' @title miaViz example data
+#'
+#' @description
+#' These example data objects were prepared to serve as examples. See the 
+#' details for more infromation.
```
Comment (microsud) [inline R/miaViz.R:27]: infromation -> information

**Review by antagomir — APPROVED** [review]

**Comment by microsud** [comment]
I get this warning when `show_label=TRUE` 
```
##Warning message:
Ignoring unknown parameters: max.overlaps 
```

**Comment by FelixErnst** [comment]
> I get this warning when `show_label=TRUE`
> 
> ```
> ##Warning message:
> Ignoring unknown parameters: max.overlaps 
> ```

I'll have a look at it.

**Comment by FelixErnst** [comment]
> *this is tricky because too many points to label. Would it be possible to have a user defined cut-off for labels? For.eg.
> if a node has more than three or four edges with weights more than a give value, that node is labelled? Something similar to the prevalence option, i.e. detection atleast 0.001 in 50% samples.

Instead of having a specific argument, you can use logical, integer and character values to subset to specific labels. I adjusted the man page regarding this behavior and included other examples.

> ##Warning message:
> Ignoring unknown parameters: max.overlaps 

I couldn't reproduce this. Which version of `ggraph` do you have installed?

**Comment by microsud** [comment]
These are the versions of key packages 

```
R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=English_Netherlands.1252  LC_CTYPE=English_Netherlands.1252   
[3] LC_MONETARY=English_Netherlands.1252 LC_NUMERIC=C                        
[5] LC_TIME=English_Netherlands.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] miaViz_0.98.8                   ggraph_2.0.3                   
 [3] ggplot2_3.3.2                   mia_0.98.31                    
 [5] TreeSummarizedExperiment_1.99.9 Biostrings_2.58.0              
 [7] XVector_0.30.0                  SingleCellExperiment_1.12.0    
 [9] SummarizedExperiment_1.20.0     Biobase_2.50.0                 
[11] GenomicRanges_1.42.0            GenomeInfoDb_1.26.0            
[13] IRanges_2.24.0                  S4Vectors_0.28.0               
[15] BiocGenerics_0.36.0             MatrixGenerics_1.2.0           
[17] matrixStats_0.57.0             

```

**Comment by FelixErnst** [comment]
Let me know, whether it works for you now or not. The argument in question is used in `geom_node_label` and hasn't changed since `2.0.3` so I have no idea, what could be wrong.

**Comment by microsud** [comment]
The warning message persisted. I think the `max.overlaps`  is passed on to the `ggrepel::geom_text_repel()`. I had ggrepel v0.8.2. Therefore, I updated ggrepl to v0.9.1 and the error is now gone. The rest is all okay including the new addition of `edge_type`.

**Review by microsud — APPROVED** [review]

**Pull request merged**